### PR TITLE
Raincatch-858

### PR DESCRIPTION
**Pull request by witmicko**
added exercise api

@paolobueno 
@wtrocki

### CODE REVIEW

**Review by paolobueno — APPROVED** [review]
LGTM, nice fixing of the linting errors too! :smile: 

A good followup would be to figure out how it would look like if `SeeFood` had a list of `Task`s in it.

i.e. `GET /api/seefood/1/tasks`, `POST /api/seefod/1/tasks` to create a task for it and so on.

**Comment by wtrocki** [comment]
> A good followup would be to figure out how it would look like if SeeFood had a list of Tasks in it.
> i.e. GET /api/seefood/1/tasks, POST /api/seefod/1/tasks to create a task for it and so on.

👍

File: apps/cloud/src/api/see_food.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import * as express from 'express';
+
+interface SeeFood {
+  id: string;
+  isHotdog: boolean;
+  mikeHuntScore: number;
+}
+
+const config: WebApiConfig = {
+  limits: true
```
Comment (wtrocki) [inline apps/cloud/src/api/see_food.ts:12]: Another follow up idea is to make config optional. Maybe we can extend it by for example specifying what CRUD operations are possible?

**Review by wtrocki — APPROVED** [review]

**Comment by paolobueno** [comment]
@witmicko, @wtrocki are we going to merge these? I thought this was mostly an exercise

**Comment by witmicko** [comment]
> are we going to merge these? I thought this was mostly an exercise

I guess we can merge those for now, this will allow people to update their local without having to stash, we'll remove those files after spike week maybe.

**Pull request merged**